msg/manual_control_setpoint.msg

```diff
@@ -34,11 +34,11 @@ float32 y			 # stick position in y direction -1..1
 				 # in general corresponds to right/left motion or roll of vehicle,
 				 # in general a positive value means right or positive roll and
 				 # a negative value means left or negative roll
-float32 z			 # throttle stick position 0..1
+float32 z			 # throttle stick position -1..1
```

Comment:
This affects log analysis (Flight Review) and thus needs to be detectable. Can you rename the field/message?

Reply:
I'm aware that it's different for log analysis but I was assuming it shows correctly and the different range in the filename can just be removed:
![image](https://user-images.githubusercontent.com/4668506/95833482-fcc60700-0d3b-11eb-9ace-94eb8003ed42.png)
Is that not good enough?
If not and I'm changing the name of the field anyways can I make it an array for all sticks axes? Like https://github.com/PX4/Firmware/blob/74c9ba8d55f3659cfbb18a82c14f3a9e64b03183/msg/vehicle_angular_velocity.msg#L5

Reply:
It's also used in the 3D view, and there might be other users we're not aware of. See https://dev.px4.io/master/en/middleware/uorb.html#deprecation.
xyz is ok.

Reply:
New docs link: https://docs.px4.io/master/en/middleware/uorb.html#message-field-deprecation

I renamed the fields to xyzr like discussed and marked the old field `DEPRECATED` like described in the docs ✔️